modules/doc/content/newsletter/2024/2024_09.md

```diff
@@ -40,8 +28,58 @@ This change should be made as soon as possible as the logic that was changed her
+or they can be set to instantiate any `Physics` onto their domain.
+
+The main difference with the Thermal Hydraulics module [Components](syntax/Components/index.md) is that `ActionComponents` are derived
+from an `Action` base class and therefore can follow the traditional task-based simulation setup, while `Components` are derived
+from `MooseObject` and require inter-component dependency resolution for the simulation setup.
+Other differences include the possibility to use `MeshGenerators` in tandem with `ActionComponents`.
+
+## MOOSE Modules Changes
+
+### Release of the fluid properties external submodules
+
+Fluid properties submodules for air, carbon dioxide, helium, nitrogen, sodium and potassium, including
+both single phase, two phase properties properties were added to MOOSE. They are optional submodules,
+they can be downloaded using the command below. Once downloaded, an application must use the fluid properties
+module to be able to use these properties.
+
+```
+cd ~/projects/moose
+git submodule update --init --checkout modules/fluid_properties/contrib/<submodule of interest>
+```
+
+### Physics for k-epsilon turbulence
+
+The [WCNSFVTurbulencePhysics.md] was added to the Navier Stokes module. It facilitates the setup of the k-epsilon
+equations in a Navier Stokes flow simulation with the weakly compressible finite volume discretization. It is currently
+only available for the Newton solver. Performance is limited for larger problems.
+A similar `Physics` should be created for the high performance linear finite volume segregated solver in the next financial year.
+
+### Specific internal energy added to Solid Properties objects
+
+The [ThermalSolidProperties.md] objects from the [Solid properties module](modules/solid_properties/index.md)
+now provide interfaces to compute the specific internal energy as a function of
+temperature. This new interface is useful, for example, in measuring the
+internal energy in solids for energy conservation verification.
+
 ## libMesh-level Changes
 
 ## PETSc-level Changes
 
 ## Bug Fixes and Minor Enhancements
+
+- linking warnings due to visibility settings differences at compile-time of various templates
+  were resolved by using template instantiation declaration in the affected headers.
+- the framework folders are now all built using the folder unity technique which speeds up the compilation.
+  As a reminder, unity builds can be disable by setting `export MOOSE_UNITY=false` to reduce memory use during compilation
+- the `execute_on` parameter was removed from `Function` objects, as they are generally never executed on
+  an execution schedule but rather computed on-the-fly on every call.
+- the [ProjectionAux.md] specifically checks for mixed order projections to avoid double-counting nodes present
+  in both lower and higher dimensional blocks
+- the [QuadraturePositions.md] object was added to output quadrature points from elements in either the mesh or
```

Comment:
```suggestion
- Linking warnings due to visibility settings differences at compile-time of various templates
  were resolved by using template instantiation declaration in the affected headers.
- The framework folders are now all built using the folder unity technique which speeds up the compilation.
  As a reminder, unity builds can be disable by setting `export MOOSE_UNITY=false` to reduce memory use during compilation
- The `execute_on` parameter was removed from `Function` objects, as they are generally never executed on
  an execution schedule but rather computed on-the-fly on every call.
- The [ProjectionAux.md] specifically checks for mixed order projections to avoid double-counting nodes present
  in both lower and higher dimensional blocks
- The [QuadraturePointsPositions.md] object was added to output quadrature points from elements in either the mesh or
```